pkg/controllers/dynakube/connectioninfo/oneagent/reconciler.go

```diff
@@ -139,9 +151,12 @@ func (r *reconciler) createTenantTokenSecret(ctx context.Context, secretName str
 	err = query.CreateOrUpdate(*secret)
 	if err != nil {
 		log.Info("could not create or update secret for connection info", "name", secret.Name)
+		conditions.SetKubeApiErrorCondition(r.dynakube.Conditions(), oaConnectionInfoConditionType, err)
 
 		return err
 	}
 
+	conditions.SetSecretCreatedCondition(r.dynakube.Conditions(), oaConnectionInfoConditionType, secret.Name+" created")
```

Comment:
may be for future, I prefer to create a list of constants with Type of ConditionType and values like `condtype.OAConnectionInfo` or something like it. So you have a small package wich has all condition types.

Reply:
I don't want every condition type to be in a singular package.

There can be common `Reasons` for a condition but the condition `Type` should be specific to the package that handles/owns that condition.

I see no reason for different packages to care about each other's conditions.